README.md

```diff
@@ -70,13 +71,10 @@ You can also post bug reports and feature requests in Github issues.
 
 
 ## Releases
-<!--
-TODO: might want to add TensorFlow compatibility information.
-- 1.0.0 release: Spark 2.3 required. Python 3.6 & Scala 2.11 recommended. TensorFlow 1.5.0+ required.
-    1. Using the definition of images from Spark 2.3. The new definition uses the BGR channel ordering 
+- [1.0.0](https://github.com/databricks/spark-deep-learning/releases/tag/v1.0.0) release: Spark 2.3.0 required. Python 3.6 & Scala 2.11 recommended. TensorFlow 1.6.0 & Keras 2.1.5 required.
```

Comment:
Do you know if the keras version is required? (or suggested)

Reply:
Tried some different version. It seems that there is not requirement for the Keras version. Removed it.